apps/web/components/BasePrice/types.ts

```diff
@@ -0,0 +1,7 @@
+export type BasePriceProps = {
+  unitContent?: number;
+  unitName?: string;
+  basePrice?: string;
+  oneline: boolean;
```

Comment:
**thought**: I know that the PBI says the component should be reusable, but looking at the props, I'm not sure that's really the way to go. The component is small (and will get even smaller without the conditions), so we could also have a `BasePrice` and a `BasePriceInline`.

My gut feeling is that with props, we should distinguish between props that supply data and props that do nothing but create conditions inside the template.

Would welcome further discussion here @plentymarkets/shop-front-end.
(Obviously this is not blocking in any way, we can refactor it later on.)

Reply:
i also thought this was a good idea but timp said he wanted 1 component with divs switching from css